lib/Doctrine/ORM/Tools/Export/Driver/PhpExporter.php

```diff
@@ -114,10 +114,11 @@ public function exportClassMetadata(ClassMetadataInfo $metadata)
 
             if ($associationMapping['type'] & ClassMetadataInfo::TO_ONE) {
                 $method = 'mapOneToOne';
+                $joinColumns = $associationMapping['isOwningSide'] === true ? $associationMapping['joinColumns'] : array();
```

Comment:
`[]` can be used here. Variable also not needed (can directly be inlined into line 121). `=== true` also not needed, as the value is boolean